pegasus/sites.v3/code.org/public/teacher-dashboard/sections.haml

```diff
@@ -116,42 +118,42 @@ content-type: text/ng-template
           %input{:type=>'hidden', :name=>'script', :value=>'{{section.script.name}}'}
           %input.btn.btn-white{type:"submit", value: I18n.t('dashboard_action_print_certificates'), style:"margin-top:10px; width:100%"}
 
-  %br/
-  %br/
+%br/
+%br/
 
-  %p{style: "font-size:18px; color: #7665a0;"}= I18n.t('dashboard_sections_table_secret')
-  %p= I18n.t('dashboard_sections_login_type_explanation')
+%p{style: "font-size:18px; color: #7665a0;"}= I18n.t('dashboard_sections_table_secret')
+%p= I18n.t('dashboard_sections_login_type_explanation')
 
-  %br/
+%br/
 
-  %div
-    %table{style: "width:100%; table-layout:fixed;"}
-      %tr
-        %th.manage-th
-        %th.manage-th= I18n.t('dashboard_login_picture')
-        %th.manage-th= I18n.t('dashboard_login_word')
-        %th.manage-th= I18n.t('dashboard_login_none')
-      %tr
-        %td{style: "font-family: 'Gotham 7r', sans-serif;"}= I18n.t('dashboard_sections_email_question')
-        %td= I18n.t('dashboard_action_delete_no')
-        %td= I18n.t('dashboard_action_delete_no')
-        %td= I18n.t('dashboard_action_delete_yes')
-      %tr
-        %td{style: "font-family: 'Gotham 7r', sans-serif;"}= I18n.t('dashboard_sections_account_creation_question')
-        %td= I18n.t('dashboard_teacher')
-        %td= I18n.t('dashboard_teacher')
-        %td= I18n.t('dashboard_student')
-      %tr
-        %td{style: "font-family: 'Gotham 7r', sans-serif;"}= I18n.t('dashboard_sections_password_question')
-        %td= I18n.t('dashboard_sections_picture_account_creation')
-        %td= I18n.t('dashboard_sections_word_account_creation')
-        %td= I18n.t('dashboard_sections_password_account_creation')
-      %tr
-        %td{style: "font-family: 'Gotham 7r', sans-serif;"}= I18n.t('dashboard_sections_sign_in_question')
-        %td= I18n.t('dashboard_sections_sign_in_unique')
-        %td= I18n.t('dashboard_sections_sign_in_unique')
-        %td= I18n.t('dashboard_sections_sign_in_main')
+%div
```

Comment:
Could argue that we should move this loginType table into react as well (just to have everything in one place). The main reason I haven't done so is that I still need to figure out the I18n story.